VelocitySDK/Support/VLTProtobufHelper.m

```diff
@@ -30,7 +29,6 @@ + (nonnull VLTPBCapture *)captureFromDatas:(nonnull NSArray<VLTData *> *)datas
     capture.ifa           = ifa;
     capture.timestamp     = [NSDate date].timeIntervalSince1970;
     capture.platform      = VLTPBPlatformType_Ios;
-    capture.sequenceIndex = seqIndex;
```

Comment:
Because it is a required field: https://github.com/velocity-inc/velocity-protobuf/blob/master/definitions/velocity.proto#L40

Probably set it to 0.

Reply:
My bad. I did took a look at `go` implementation (was checking ingress project if its safe to remove this). And that implementation implied that this is not required:

```
func (m *Capture) GetSequenceIndex() uint32 {
	if m != nil && m.SequenceIndex != nil {
		return *m.SequenceIndex
	}
	return 0
} 
```

Reply:
👍